sos/report/plugins/hyperv.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Hyperv(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Hyper-V client information"""
+
+    plugin_name = "hyperv"
+    files = ('/sys/bus/vmbus/', '/boot/config-*')
```

Comment:
Activating on `/sys/bus/vmbus` is fine (as far as I can tell anyways), but specifying `/boot/config-*` here will enable this on literally every modern Linux installation.